Source/WebCore/page/FrameView.cpp

```diff
@@ -2401,6 +2407,12 @@ void FrameView::scheduleScrollToFocusedElement(SelectionRevealMode selectionReve
     m_delayedScrollToFocusedElementTimer.startOneShot(0_s);
 }
 
+void FrameView::cancelScheduledScrolls()
+{
+    cancelScheduledScrollToFocusedElement();
```

Comment:
What happens when this is canceled but the other isn't? It seems a bit fragile that this relies on every call site of this function to be also calling the other function.

If we should always be calling the new function instead, perhaps we should make this private.

Reply:
The whole point of this function was to tie these together, as Tim suggested. I'm unsure if there is a reason to cancel one and not the other, so I did not go so far as to make it private.

Reply:
(but I think you can make the two functions it *calls* private)